pom.xml

```
@@ -59,13 +59,14 @@
         <mssql-jdbc.version>10.2.0.jre8</mssql-jdbc.version>
         <ojdbc8.version>21.5.0.0</ojdbc8.version>
         <mongodb-driver.version>4.6.1</mongodb-driver.version>
-        <clickhouse.version>1.4.4</clickhouse.version>
+        <clickhouse.version>1.${commons-collections4.version}</clickhouse.version>
```

```suggestion
        <clickhouse.version>1.4.4</clickhouse.version>
```
why use ${commons-collections4.version} instead of clickhouse version?